README.md

```diff
@@ -1,4 +1,7 @@
-# QuDynamics.jl
+QuDynamics.jl
+============
+
+[![Build Status](https://travis-ci.org/JuliaQuantum/QuDynamics.jl.png?branch=master)](https://travis-ci.org/JuliaQuantum/QuDynamics.jl)
```

Comment:
Yeah, seems you can specify for which branch you want to show the status. So I guess this is fine then.